crt_portal/cts_forms/tests.py

```diff
@@ -109,3 +87,62 @@ def test_required_where(self):
             'public_or_private_healthcare': 'state_local_facility',
         })
         self.assertTrue('place<ul class="errorlist"><li>This field is required.' in str(form.errors))
+
+
+class ContactValidationTests(TestCase):
+    def test_non_ascii_name(self):
+        form = Contact(data={
+            'contact_first_name': '李王',
+            'contact_last_name': '王-Núñez',
+            'contact_email': '',
+            'contact_phone': ''
+        })
+        self.assertTrue(form.is_valid())
+
+    def test_non_ascii_email(self):
+        form = Contact(data={
+            'contact_first_name': '',
+            'contact_last_name': '',
+            'contact_email': 'foo@bär.com',
+            'contact_phone': ''
+        })
+        self.assertTrue(form.is_valid())
+
+    def test_invalid_phone(self):
+        form = Contact(data={
+            'contact_first_name': '',
+            'contact_last_name': '',
+            'contact_email': 'foo@bär.com',
+            'contact_phone': '33333333333333333333333333'
+        })
+        self.assertFalse(form.is_valid())
```

Comment:
I like testing for the correct error. In this case,
` self.assertTrue(phone_error_message == ['Enter a valid value.'])`
because then we are checking for the right error and won't get noise or miss something if there is a different error,

Reply:
There's a test for the correct error here: https://github.com/usdoj-crt/crt-portal/pull/52/files/5bb2f65a481b18f7165e7ecbc9ba98ff82519e5d#diff-18509124e855af21de52f53726ce2005R130.